src/main/scala/org/broadinstitute/dsde/firecloud/model/Workspace.scala

```diff
@@ -24,6 +23,15 @@ object WorkspaceCreate {
   def isProtected(ws: Workspace): Boolean = {
     ws.realm == Some(RawlsRealmRef(RawlsGroupName(FireCloudConfig.Nih.rawlsGroupName)))
   }
+
+  def toWorkspaceClone(wc: WorkspaceCreate): WorkspaceCreate = {
+    new WorkspaceCreate(
+      namespace = wc.namespace,
+      name = wc.name,
+      attributes = wc.attributes + (AttributeName("library","published") -> AttributeBoolean(false)),
+      isProtected = wc.isProtected)
+  }
```

Comment:
I think
```
    wc.attributes - AttributeName("library","published")
```
would be better - a missing library:published attribute defaults to false, and is semantically equivalent to the workspace having never been published.

'course, that one-line change would wreak havoc on your unit tests :(